app/stylesheets/boilerplate/view-mode.scss

```diff
@@ -1,12 +1,38 @@
 @import '../config/colours';
 @import '../config/list';
+@import '../config/dimensions';
+@import '../config/type';
 
-.view-mode {
-  @include list-inline();
+.view-mode-wrapper {
+  display: table-cell;
+  vertical-align: top;
+  width: 100%;
+}
+
+ul.view-mode {
   @include list-unstyle();
-  background: $white;
+  background: $bg-color;
+  display: table;
+  float: right;
+  table-layout: fixed;
+  width: 100%;
 
   li {
```

Comment:
Lets do `& > li` here. (I know, I wrote this li selector I'm now complaining about :smile: )